initialize.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+set -e
+
+MINIMUM_DOCKER_VERSION=1.13.0
+MINIMUM_DOCKER_COMPOSE_VERSION=1.23.2
+MINIMUM_PYTHON_VERSION=3.6
+
+# Function to compare 2 semver version
+semantic_version_comp () {
+    if [[ $1 == $2 ]]
+    then
+        echo 0
+        return
+    fi
+    local IFS=.
+    local i ver1=($1) ver2=($2)
+    # fill empty fields in ver1 with zeros
+    for ((i=${#ver1[@]}; i<${#ver2[@]}; i++))
+    do
+        ver1[i]=0
+    done
+    for ((i=0; i<${#ver1[@]}; i++))
+    do
+        if [[ -z ${ver2[i]} ]]
+        then
+            # fill empty fields in ver2 with zeros
+            ver2[i]=0
+        fi
+        if ((10#${ver1[i]} > 10#${ver2[i]}))
+        then
+            echo 1
+            return
+        fi
+        if ((10#${ver1[i]} < 10#${ver2[i]}))
+        then
+            echo 2
```

Comment:
would it be better to use a string instead of an integer for better clarity?

Reply:
@mlodic "eq", "lt", "gt" instead of 1,2,3?

Reply:
the full word even better I guess

Reply:
done